modules/experimental-layers/src/addons/json-deck/shallow-equal.js

```diff
@@ -0,0 +1,28 @@
+// TODO - can we use the core util?
+
+/* eslint-disable complexity */
+export function shallowEqual(a, b, {ignore = {}} = {}) {
```

Comment:
> We already have several comparison utils in core, including this one:

Yes, this was copied from core. Question is of course whether we want to export such utility functions from core. Since json is expected to go into a separate module, I copied it for now.

> These shallowEqual and deepEqual functions don't actually do what they claim to. At least name them something more meaningful?

Any suggestions? This function does shallowly compare keys between two objects?